docs/user/setup.rst

```diff
@@ -93,16 +93,21 @@ installation of Arch Linux.
     $ sudo dnf groupinstall "Development Tools"
     $ sudo dnf install gc-devel zlib-devel # both optional
 
-**FreeBSD 12.2 and later**
+**FreeBSD 12.4 and later**
 
 .. code-block:: shell
 
-    $ pkg install llvm10
+    $ pkg install llvm # optional
     $ pkg install boehm-gc # optional
 
-*Note:* A version of zlib that is sufficiently recent comes with the
+*Note 1:* Sufficiently recent versions of llvm and zlib come with the
```

Comment:
Perhaps move "boehm-gc" line after this paragraph,
then remove "Note" verbiage and promote to full paragraph? Then notes become renumbered.

Reply:
I'm not sure to fully understand how you'd like that I change it.

Reply:
I think this simplifies down to something like:

```
**FreeBSD 12.4 and later**
*Note 1:* Only AMD64 and ARM64 architectures are supported.
*Note 2:* Sufficiently recent versions of llvm and zlib come with the
installation of FreeBSD.

.. code-block:: shell

    $ pkg install boehm-gc # optional

*Note 3:* Using the boehm GC with multi-threaded binaries doesn't work
out-of-the-box yet.
```

Puts the important information (only 64 bit) first and
keeps optional  boehm-gc info close together in layout.

What do you think?  Feel free to reject or wordsmith.